javascript/javascript1/week4/js1w4-hw.js

```diff
@@ -0,0 +1,137 @@
+// VOICE ASSISTANT COMMAND HOME WORK
+
+let string = "Hello my name is Benjamin";
+let stringArray = string.split(" ");
+let name = stringArray[stringArray.length - 1];
+let names = [];
+let whoAmI = "What is my name";
+let day = "What day is it today";
+let todo = [];
+let myTodoList = "What is on my todo";
+let favDish = [];
+let event = [];
+let eventQuestion = "What am I doing this week";
+
+function getReply(command) {
+  //finding it hard to make this statement work
+  if (command === string) {
+    for (let i = 0; i < names.length; i++) {
+      if (command === whoAmI && names[i] === names[i]) {
```

Comment:
you already tested `command===string` above, so `command===whoAmI` will always be false in this part of the code.